### ISSUE TITLE
Add TERMINALVELOCITY

### ISSUE THREAD

**Issue by MOARdV** [Mihara/RasterPropMonitor#255]
Per MechJeb, it's

```
double airDensity = FlightGlobals.getAtmDensity(FlightGlobals.getStaticPressure(CoM, mainBody));
return Math.Sqrt(2 * localg * mass / (massDrag * FlightGlobals.DragMultiplier * airDensity));
```

**Comment by MOARdV** [comment]
Oh, and for the massDrag variable:

    foreach (Part p in vessel.parts)
    {
        if (p.rb != null)
        {
            massDrag += p.rb.mass * p.maximum_drag;
        }
    }

**Comment by MOARdV** [comment]
Now available for v0.21.0
